app/assets/stylesheets/custom.scss

```diff
@@ -137,3 +137,18 @@ a {
 td {
   background-color: rgba(255, 255, 255, .5);
 }
+
+// variables, mixins
+@mixin box-sizing {
+  box-sizing: border-box;
+}
+
+// miscellaneous
+
+.debug-dump {
+  clear: both;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets